components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1056,7 +1063,14 @@ public MetadataStore getMetadataStoreForConversion() {
     int realSeries = getSeries();
     for (int i=0; i<getSeriesCount(); i++) {
       setSeries(i);
-      store.setPixelsBinDataBigEndian(new Boolean(!isLittleEndian()), i, 0);
+      if (meta.getPixelsBinDataCount(i) > 0) {
+        for (int j=0; j<meta.getPixelsBinDataCount(i); j++) {
+          store.setPixelsBinDataBigEndian(new Boolean(!isLittleEndian()), i, j);
+        }
+      }
+      else {
+        store.setPixelsBigEndian(Boolean.TRUE, i);
```

Comment:
From [our specification](http://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Pixels_BigEndian0), `Pixels.BigEndian` can be optional and should definitely match the value of `BinData.BigEndian` if the latter exists i.e. `!isLittleEndian()`. No strong objection to setting it unconditionally.